### PR TITLE
Add endpoint for getting reservation item info

### PR DESCRIPTION
**Pull request by FrNecas**
Will be useful for re-fetching item expiration when a manager on FE extends it without having to fetch the whole reservation

### CODE REVIEW

File: KachnaOnline.App/Controllers/BoardGamesReservationsController.cs [hunk 1]
```diff
@@ -116,6 +116,34 @@ public async Task<ActionResult<ReservationDto>> GetReservation(int id)
...
+        /// Returns an item of a reservation with the given ID.
+        /// </summary>
+        /// <param name="id">ID of the reservation which the item is in.</param>
+        /// <param name="itemId">ID of the item to return.</param>
+        /// <response code="200">The reservation item.</response>
+        /// <response code="403">The user is not a board games manager and the reservation belongs to another user.</response>
+        /// <response code="404">No such reservation item exists.</response>
+        [HttpGet("{id}/{itemId}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
```
Comment (Adda0) [inline KachnaOnline.App/Controllers/BoardGamesReservationsController.cs:129]: Considering 403 should not be explicitly states as a response code, I would consider removing both 403 lines from here. See @ondryaso comment on #23:
> Explicitly stating the 403 response code is redundant and inconsistent.

**Review by Adda0 — CHANGES_REQUESTED** [review]
LGTM otherwise.

File: KachnaOnline.App/Controllers/BoardGamesReservationsController.cs (hunk 1 above)
Comment (FrNecas) [reply KachnaOnline.App/Controllers/BoardGamesReservationsController.cs:129]: I think we discussed that this is a valid thing, same as with `GetReservation` but I may be wrong. The thing is that it doesn't have the regular semantic "you don't have access to this endpoint overall" but rather "you don't have access to an endpoint with these particular parameters"

Comment (Adda0) [reply KachnaOnline.App/Controllers/BoardGamesReservationsController.cs:129]: I agree to this, as it seems reasonable to have it explicitly stated. What does @ondryaso think? In that case, I approve the PR and leave it to you.

**Review by Adda0 — APPROVED** [review]
As stated above, LGTM otherwise.

**Pull request merged**